.github/workflows/ci.yml

```diff
@@ -15,3 +18,6 @@ jobs:
 
     - name: readme
       run: make readme-all
+
+    # Commit all changed files back to the repository (dist and readme files)
+    - uses: stefanzweifel/git-auto-commit-action@v4
```

Comment:
What does this commit back to the repo?

Reply:
This is the result of `npm run build` and the potential updates to the readme files.
(Previously this was done local)

Reply:
Since this is just a `push` workflow, what stops this extra commit from triggering the workflow again? Should we be setting the commit message to something useful (or even including the skip-ci directive)?